src/ApiRequestEditorElement.js

```diff
@@ -731,6 +731,23 @@ export class ApiRequestEditorElement extends AmfHelperMixin(EventsTargetMixin(Li
     this.abort();
   }
 
+  requestHeaders() {
```

Comment:
Why do you want to generate the `boundary` here? There's no point. The transport library (XHR/Fetch) does this. API Console should not do this.

Reply:
The library is not generating the boundary and that's why I had to generate it before making the request. I've read in some posts about how to workaround this issue, and the options are: generating it like I did or explicitly set Content-type to undefined.

Reply:
Let's discuss this on a call. I have to understand the reason why we would need this :)